bedrock/firefox/views.py

```diff
@@ -403,6 +402,7 @@ class WhatsnewView(L10nTemplateView):
         "firefox/whatsnew/whatsnew-fx126beta-fr.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx126-na.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx126-eu.html": ["firefox/whatsnew/whatsnew"],
+        "firefox/whatsnew/whatsnew-fx126-mobile.html": ["firefox/whatsnew/whatsnew-account", "firefox/whatsnew/whatsnew"],
```

Comment:
**Question:** is this unrelated to this PR? I don't see this template anywhere?

Reply:
This began life as a proposed experiment for 126, but we decided not to do that and just make this the sole fallback so I kind of hastily moved things around and clearly missed a few spots. I should have been more careful.